torch/_export/converter.py

```diff
@@ -194,6 +283,11 @@ def convert_graph_inputs(self):
                         target=name,
                     )
                 )
+                fx_node = self.fx_graph.placeholder(normalized_name)
+
+            # TODO: set fx_node.meta["val"]
```

Comment:
do we need to set this? It should get automatically populated for the exported program's graph through retracing...

Reply:
I think we still need those for the top level module?

Reply:
I'm referring to the `TODO: set fx_node.meta["val"]`. I don't think this needs to be done in the converter?